src/classes/UTIL_MasterSchedulableHelper.cls

```diff
@@ -185,4 +185,22 @@ public without sharing class UTIL_MasterSchedulableHelper {
             ERR_Handler.processError(ex, ERR_Handler_API.Context.STTG);
         }
     }
+
+    /*******************************************************************************************************
+    * @description Checks for presently running jobs of a particular method
+    * @param methodName The name of the method to check for running jobs on
+    * @return Boolean
+    */
+    public static Boolean hasRunningJob(String className) {
+        Integer jobCount = [SELECT COUNT()
+            FROM AsyncApexJob
+            WHERE ApexClass.Name = :className
```

Comment:
I was thinking about this the other day. The `ApexClass` object has a separate field for `NamespacePrefix`, which means I wasn't quite right about the query failing in a namespaced org. The `Name` field is the name of the class without the namespace. I still prefer the syntax in the classes to get the full name of the class (ex: `CRLP_RDSkew_BATCH.class.getName()`) rather than hardcoding the classname as a string, but the method here will need a slight change to remove the namespace from the classname before the query. I suggest something like the following:
```className = (className.contains('__') ? className.split('__')[1] : className);```

Also, wouldn't hurt to test this out in a namespaced scratch org to verify that the logic works as expected with or without the namespace.